bindings/py/packaging/setup.py

```diff
@@ -340,7 +347,13 @@ def generateExtensions(platform, build_type):
         "htm.bindings": ["*.so", "*.pyd"],
         "htm.examples": ["*.csv"],
     },
-    extras_require = {},
+    #install extras by `pip install htm.core[examples]`
+    extras_require={'scikit-image>0.15.0':'examples',
```

Comment:
split requirements.txt to hard-core dependencies (requirements.txt) , and optional (in setup.py extras_required)
This is due to Arm64 which failed on some of the new pip dependencies (scikit ?)